src/community/community.ts

```diff
@@ -146,18 +146,21 @@ export class Community extends EventEmitter {
 }
 
 /**
- * This waits until the libp2p node has connected to two peers
+ * This waits until the libp2p node has connected to a signer
  * @private
  */
-export function afterThreePeersConnected(node:IP2PNode):Promise<void> {
+export async function afterOneSignerConnected(node:IP2PNode, group:NotaryGroup):Promise<void> {
```

Comment:
In briefly starting down this road, I was considering having this waiter take an optional `minPeers` so you could have stronger about the connection and where messages are being sent. Happy to do that as an additional PR, but that might be yagni?